### PR TITLE
Rebuild, new try

### PR DESCRIPTION
**Pull request by cbrueffer**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by anjakefala** [comment]
Eyo! :smile: 

Also, apparently: https://github.com/conda-forge/conda-smithy/issues/867#issuecomment-410776205

The py<34 is what has been causing the intermittent issues. So if I change that down the line, this should theoretically work with the automated renderer. (And more consistently work with the local renderer.)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   script: python -m pip install --no-deps --ignore-installed .
   skip: True  # [py<34]
   skip: True  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: I would suggest dropping the 2 skip lines above and adding `noarch: python` if it is noarch compatible.

Comment (anjakefala) [reply recipe/meta.yaml]: It is not noarch compatible, sadly. It is not supported on Windows or python2.

It was suggested that I change the py<34 to just skipping py2.

**Comment by anjakefala** [comment]
I am going to further investigate the skip lines for future releases, but have merged this in for now so we can get a build going. Many thanks to @cbrueffer :tada: and to the conda-forge team for helping me out with this so promptly.

**Comment by cbrueffer** [comment]
Thanks for the quick iteration @anjakefala !

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: You can skip python2 with noarch, but you can't skip windows with noarch, but is it necessary to skip windows? It's not working in windows, so we can ignore it.

Comment (anjakefala) [reply recipe/meta.yaml]: oh, it would just make the build and then fail? But the failure would not stop the merging and deployment?

Comment (isuruf) [reply recipe/meta.yaml]: Noarch packages are built only on Linux, but installable on all platforms and any python version

Comment (anjakefala) [reply recipe/meta.yaml]: Awesome. Thanks isuruf! I will update it.

Comment (anjakefala) [reply recipe/meta.yaml]: How do I skip python 2 with noarch: https://github.com/conda-forge/visidata-feedstock/pull/6#issuecomment-410796149

**Pull request merged**